spacecmd/src/bin/spacecmd

```diff
@@ -35,10 +36,6 @@ except ImportError:
 import codecs
 import locale
 import argparse
-try: # python 3
-    from configparser import SafeConfigParser
-except ImportError: # python 2
-    from ConfigParser import SafeConfigParser
```

Comment:
I think we cannot break the Python2 compatibility for this particular package, as "spacecmd" is also shipped in the Client Tools channels, meaning it could run on i.a. CentOS7 client or SLE12, which uses Python 2.